apiary_v2.apib

```diff
@@ -1807,12 +2081,143 @@ This endpoint can be used to retrieve a paginated list of the individual jobs as
 
     + Body
 
-          {}
+          {
+            "count": 99,
+            "next": "https://partner.embed.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4/jobs?page=2",
+            "previous": null,
+            "results": [
+              {
+                "id": "e3dc59c3-d4ed-42bc-8d73-23dd0e5c4301",
+                "short_id": "E3DC59C3",
+                "reference": "15572690633029",
+                "product_name": "Technical Tee",
+                "product_type": "Technical Tee 313133 XL",
+                "product_code": "335644",
+                "state": "in_production",
+                "created": "2024-07-29T13:28:22.516837Z",
+                "partner_order_reference": "my-order-ref",
+                "partner_item_reference": "my-item-ref",
+                "price_per_item": "25.00",
+                "currency": "USD",
+                "size": "XL",
+                "manufacturing_template_code": "335644",
+                "quantity": 1,
+                "arrive_by_date": "2024-08-19",
+                "ex_factory_date": "2024-08-05",
+                "design_source_id": "6489daff-b713-452c-b103-c724d307cb35",
+                "partner_data": {
+                  "internal_ref": "15572690633029",
+                  "UNMADE_SOURCE_DESIGN_ID": "6489daff-b713-452c-b103-c724d307cb35",
+                  "UNMADE_ORDER_ITEM_DESIGN_ID": "34856d12-0099-4087-b71d-0517bb0e4d7e"
+                },
+                "manufacturing_order": "https://factory.unmade.com/v2/orders/a882ac66-9374-4910-8c93-9ea45a470bd4/",
+                "manufacturing_files_zip": "https://factory.unmade.com/v2/jobs/e3dc59c3-d4ed-42bc-8d73-23dd0e5c4301/manufacturing_files_zip/",
+                "ticket": "https://factory.unmade.com/v22/jobs/e3dc59c3-d4ed-42bc-8d73-23dd0e5c4301/ticket_pdf/",
+                "supplier": {
+                  "name": "Factory A",
+                  "slug": "factory-a"
```

Comment:
do we need the slug? doesn't hurt

Reply:
This is just what's in the job endpoint currently. Seems fine to me